src/vm/analysis/type_checker/tests/mod.rs

```diff
@@ -216,7 +216,7 @@ fn test_eqs() {
 
 #[test]
 fn test_lists() {
-    let good = ["(map hash160 (list 1 2 3 4 5))",
+    let good = ["(map hash160 (list u1 u2 u3 u4 u5))",
```

Comment:
Can you preserve the test signed integers here as well?

Reply:
Addressed with https://github.com/blockstack/blockstack-core/pull/1129/commits/50149b7a553118e70688d3cd2ea47e9e516683c4